routers/reviewer.router.js

```diff
@@ -0,0 +1,26 @@
+const reviewersController = require('../controllers/reviewers.controller').userController;
```

Comment:
We can park the feature of `reviewer` as a role under the user resource. Consider deleting all related files to `reviewers`.